WordPress/Classes/Login/LoginWithUrlView.swift

```diff
@@ -47,31 +60,45 @@ struct LoginWithUrlView: View {
 
             Spacer()
 
-            Button(action: startLogin, label: {
-                HStack(alignment: .center) {
-                    Spacer()
-                    if isLoading {
-                        ProgressView()
-                    } else {
-                        Text("Continue")
+            let button = Button(action: startLogin) {
+                    HStack(alignment: .center) {
+                        Spacer()
+                        if isLoading {
+                            ProgressView()
+                        } else {
+                            Text(SharedStrings.Button.continue)
+                        }
+                        Spacer()
                     }
-                    Spacer()
+                    .padding(8)
                 }
-                .animation(.default, value: 0)
-                .padding()
-                .background(
-                    RoundedRectangle(
-                        cornerRadius: .DS.Radius.small,
-                        style: .continuous
-                    )
-                    .stroke(.primary, lineWidth: 2)
-                )
-            })
-            .tint(.primary)
-        }.padding()
+                .disabled(isContinueButtonDisabled)
+
+            if isContinueButtonDisabled {
+                button.buttonStyle(.bordered)
```

Comment:
This button doesn't look quite right. I suggest using the existing `DSButton` for now until there is a better replacement.

```swift
DSButton(title: SharedStrings.Button.continue, style: DSButtonStyle(emphasis: .primary, size: .large), isLoading: isLoading) {
    // TODO: handle action
}
.disabled(isContinueButtonDisabled)
```

 A better replacement should be implemented using native `ButtonStyle` API so you could use it as a modifier. I added the following type recently but it doesn't support the full-width presentation yet:

```swift
public struct PrimaryButtonStyle: ButtonStyle {
```